docs/providers/aws/guide/functions.md

```diff
@@ -203,6 +203,8 @@ See the documentation about [IAM](./iam.md) for function level IAM roles.
 
 Alternatively lambda environment can be configured through docker images. Image published to AWS ECR registry can be referenced as lambda source (check [AWS Lambda – Container Image Support](https://aws.amazon.com/blogs/aws/new-for-aws-lambda-container-image-support/)). In addition, you can also define your own images that will be built locally and uploaded to AWS ECR registry.
 
+In service configuration, you can configure the ECR image repository to scan for CVEs via the `provider.ecr.scanOnPush` property. (See [documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html))
```

Comment:
It would be great to highlight that this property will only be applied for new ECR repositories being created so either for new services or the first time a function configured with `image` is deployed. Unfortunately, ECR repo is not maintained via CloudFormation at the moment and we don't have a mechanism to update it.

Reply:
Gotcha, that's a good point! I'll update it to something. As far as the test goes, do you have suggestions? Maybe mocking out the aws-sdk call? Do we do this in other places? Thanks @pgrzesik